Add missing MSG_CMSG_CLOEXEC

Unpatched uClibc toolchains, even using the latest 0.9.33.2, do not
have the MSG_CMSG_CLOEXEC definition. Even though the Buildroot
internal toolchain backend has a uClibc patch to provide it, it
doesn't apply to external toolchains. This patch provides the
definition of MSG_CMSG_CLOEXEC.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/libnetlink.h
===================================================================
--- a/libnetlink.h
+++ b/libnetlink.h
@@ -7,6 +7,10 @@
 #include <linux/netlink.h>
 #include <linux/rtnetlink.h>
 
+#ifndef MSG_CMSG_CLOEXEC
+#define MSG_CMSG_CLOEXEC 0x40000000
+#endif
+
 struct rtnl_handle
 {
 	int			fd;
Index: b/kacpimon/libnetlink.h
===================================================================
--- a/kacpimon/libnetlink.h
+++ b/kacpimon/libnetlink.h
@@ -7,6 +7,10 @@
 #include <linux/netlink.h>
 #include <linux/rtnetlink.h>
 
+#ifndef MSG_CMSG_CLOEXEC
+#define MSG_CMSG_CLOEXEC 0x40000000
+#endif
+
 struct rtnl_handle
 {
 	int			fd;
